src/pages/guide/js/index.md

```diff
@@ -0,0 +1,312 @@
+            expect(model.isSingle(0)).toBeFalsy();
+            expect(model.isMultiple(0)).toBeTruthy();
+        });
+
+        it('Check isActionVisible function', function () {
+            expect(model.isActionVisible(action)).toBeFalsy();
+            action.hidden = false;
+            expect(model.isActionVisible(action)).toBeTruthy();
+        });
+    });
+});
+```
+
+This topic doesn't provide Jasmine test writing methodology.
+
+[Learn more about testing with Jasmine.]
+
+## Debug tests
+
+Jasmine tests can be debugged in a browser using the following steps:
+
+To keep the webserver running, set `keepalive` setting to `true` in the `dev/tests/js/jasmine/spec_runner/settings.json` file.
+
+Launch the tests with the `grunt spec:luma` CLI command. Now the webserver should be started and waiting, `_SpecRunner.html` file should be generated in the project root.
+
+Go to `http://localhost:8000/_SpecRunner.html` and use the developer console to debug the tests.
+
+The array of the tests can be edited in the `_SpecRunner.html` file to include only necessary files.
+
+## Known issues and solutions
+
+### Error: Cannot find module '&lt;module&gt;'
+
+#### Issue:
+
+An error message appears:
+
+```terminal
+Loading "Gruntfile.js" tasks...ERROR
+
+>> Error: Cannot find module '<module>'
+
+Warning: Task "spec" not found. Use --force to continue.
+```
+
+#### Solution:
+
+1. Make sure your Node.js version is up-to-date.
+1. Remove `package.json`, `Gruntfile.js`.
+1. Copy `package.json`, `Gruntfile.js` from `package.json.sample`, `Gruntfile.js.sample`.
+1. Delete the `node_modules` directory.
+1. Run `npm install` in your terminal.
+
+### Warning: Cannot read property 'pid' of undefined
+
+#### Issue:
+
+An error message appears:
+
+```terminal
+Warning: Cannot read property 'pid' of undefined
+
+Use --force to continue. Aborted due to warnings.
+```
+
+#### Solution:
+
+Run in your terminal:
+
+```bash
+cd <magento_root>/node_modules/grunt-contrib-jasmine
+```
+
+```bash
+npm install
+```
+
+<!-- LINK DEFINITIONS -->
+
+<!-- External -->
+[`<magento_root_dir>/app/code/Magento/Ui/view/base/web/js/grid/columns/actions.js`]: https://github.com/magento/magento2/blob/2.4/app/code/Magento/Ui/view/base/web/js/grid/columns/actions.js
+
+[`app/code/Magento/Ui/base/js/grid/columns/actions.test.js`]: https://github.com/magento/magento2/blob/2.4/dev/tests/js/jasmine/tests/app/code/Magento/Ui/base/js/grid/columns/actions.test.js
+
+[Deploy static view files]: https://experienceleague.adobe.com/docs/commerce-operations/configuration-guide/cli/static-view/static-view-file-deployment.html
+
+[fontconfig library]: https://www.freedesktop.org/wiki/Software/fontconfig/
+
+[Grunt]: https://gruntjs.com/
+
+[Install grunt-cli]: https://gruntjs.com/getting-started
+
+[Install Node.js]: https://nodejs.org/en/
+
+[Learn more about testing with Jasmine.]: https://jasmine.github.io/edge/introduction.html
+
+<!-- Abbreviations -->
+
+*[FTF]: Functional Testing Framework
```

Comment:
Everything below `Abbreviations` appears on the page.

Reply:
Good catch. Removed.